docs/reference/changelog-r2020.md

```diff
@@ -25,6 +26,7 @@ Released on XXX.
     - Fixed color of the bounding objects remaining in the collision state if the collision was lasting only one step (thanks to Acwok).
     - Fixed Matlab API.
     - Fixed missing stdout/stderr flush when a controller is changed or restarted while simulation is running (thanks to tsampazk).
+    - Fixed the ROS controller of the Universal Robots UR3e, UR5e and UR10e to send the success state when a trajectory is succeeded (thanks to Tim-Stoll).
```

Comment:
```suggestion
    - Fixed the ROS controller of the Universal Robots UR3e, UR5e and UR10e to send the success state when a trajectory is successful (thanks to Tim-Stoll).
```

Reply:
The ROS terminology is quite precise, so I would like to keep the succeeded word. Would this be fine:
```suggestion
    - Fixed the ROS controller of the Universal Robots UR3e, UR5e and UR10e to send the success state when a trajectory succeeded (thanks to Tim-Stoll).
```